src/unix/linux_like/linux/musl/b32/arm/mod.rs

```diff
@@ -15,35 +16,22 @@ s! {
         pub st_size: ::off_t,
         pub st_blksize: ::blksize_t,
         pub st_blocks: ::blkcnt_t,
-        pub st_atime: ::time_t,
-        pub st_atime_nsec: ::c_long,
-        pub st_mtime: ::time_t,
-        pub st_mtime_nsec: ::c_long,
-        pub st_ctime: ::time_t,
-        pub st_ctime_nsec: ::c_long,
+        __st_atime32: ::c_long,
+        __st_atime32_nsec: ::c_long,
+        __st_mtime32: ::c_long,
+        __st_mtime32_nsec: ::c_long,
+        __st_ctime32: ::c_long,
+        __st_ctime32_nsec: ::c_long,
         pub st_ino: ::ino_t,
-    }
-
-    pub struct stat64 {
-        pub st_dev: ::dev_t,
-        __st_dev_padding: ::c_int,
-        __st_ino_truncated: ::c_long,
-        pub st_mode: ::mode_t,
-        pub st_nlink: ::nlink_t,
-        pub st_uid: ::uid_t,
-        pub st_gid: ::gid_t,
-        pub st_rdev: ::dev_t,
-        __st_rdev_padding: ::c_int,
-        pub st_size: ::off_t,
-        pub st_blksize: ::blksize_t,
-        pub st_blocks: ::blkcnt_t,
         pub st_atime: ::time_t,
         pub st_atime_nsec: ::c_long,
+        __st_atime_nsec_padding: ::c_long,
         pub st_mtime: ::time_t,
         pub st_mtime_nsec: ::c_long,
+        __st_mtime_nsec_padding: ::c_long,
         pub st_ctime: ::time_t,
         pub st_ctime_nsec: ::c_long,
-        pub st_ino: ::ino_t,
+        __st_ctime_nsec_padding: ::c_long,
```

Comment:
I believe the padding here is only considering little endian.
The big endian padding happens in between `atime` and `atime_nsec` as it's defined here:
https://git.musl-libc.org/cgit/musl/tree/include/alltypes.h.in#n44

Cc @tgross35